core/quivr_core/rag/quivr_rag.py

```diff
@@ -32,6 +33,7 @@
 )
 
 logger = logging.getLogger("quivr_core")
+langfuse_handler = CallbackHandler()
```

Comment:
This is a global object cant we put it in some class?

Reply:
Like this ?

```suggestion
langfuse_service = LangfuseService()
langfuse_handler = langfuse_service.get_handler()

```

With a class defined in utils :
```
class LangfuseService:
    def __init__(self):
        self.langfuse_handler = CallbackHandler()

    def get_handler(self):
        return self.langfuse_handler
```